plugins/imdiag/imdiag.c

```diff
@@ -264,18 +264,18 @@ injectMsg(uchar *pszCmd, tcps_sess_t *pSess)
 {
 	uchar wordBuf[1024];
 	int iFrom, nMsgs;
-	uchar *litteralMsg;
+	uchar *literalMsg;
 	int i;
 	ratelimit_t *ratelimit = NULL;
 	DEFiRet;
 
-	litteralMsg = NULL;
+	literalMsg = NULL;
 
 	memset(wordBuf, 0, sizeof(wordBuf));
 	CHKiRet(ratelimitNew(&ratelimit, "imdiag", "injectmsg"));
 	/* we do not check errors here! */
 	getFirstWord(&pszCmd, wordBuf, sizeof(wordBuf), TO_LOWERCASE);
-	if (ustrcmp(UCHAR_CONSTANT("litteral"), wordBuf) == 0) {
```

Comment:
I can't tell if this is an external API or a private thing for testing.

Reply:
I've moved this to https://github.com/rsyslog/rsyslog/commit/e6764a021ca6573f71dbcdfd9d45db229676c874